app/src/main/java/com/github/houseorganizer/houseorganizer/NavBar/NavBarHelpers.java

```diff
@@ -27,6 +28,10 @@ public static Intent changeActivityIntent(String buttonText, DocumentReference c
                 Intent intentG = new Intent(context, GroceriesActivity.class);
                 intentG.putExtra("house", currentHouse.getId());
                 return intentG;
+            case "Billsharer":
+                Intent intentB = new Intent(context, ExpenseActivity.class);
+                intentB.putExtra("house", currentHouse.getId());
```

Comment:
Current house can be obtained from SharedPreferences just like in MainScreenActivity. I think it is a better practice but this solution can work too

Reply:
right i'll try that